MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include CONTRIBUTORS.txt
```

Comment:
I've checked some open source libraries and they're normally included. For instance, Django also does it: https://github.com/django/django/blob/master/MANIFEST.in